sklearnex/dispatcher.py

```diff
@@ -90,6 +91,15 @@ def get_patch_map_core(preview=False):
                     None,
                 ]
             ]
+
+            # Ridge
+            linear_model_module, _, _ = mapping["ridge"][0][0]
+            sklearn_obj = mapping["ridge"][0][1]
```

Comment:
To my best understanding so far - it is used as fallback object when sklearnex functionality fails. daal4py dispatcher fills it with stock sklearn object when patching happens and replaces sklearnex one when unpatch is needed. Since we are handling previews "manually" in sklearnex dispatcher, need to put the stock sklearn object there so during possible unpatch it knows what to fallback to.

Reply:
> To my best understanding so far - it is used as fallback object when sklearnex functionality fails. daal4py dispatcher fills it with stock sklearn object when patching happens and replaces sklearnex one when unpatch is needed. Since we are handling previews "manually" in sklearnex dispatcher, need to put the stock sklearn object there so during possible unpatch it knows what to fallback to. @icfaust please correct me if I am wrong.

You are right, to be honest, when we patch we probably should be more careful, there is definitely some concurrency bugs which maybe occurring there that we aren't testing.  At least this reasonably handles preview when another patch may be in place.